.travis.yml

```diff
@@ -20,6 +20,9 @@ matrix:
 # These directories are cached to S3 at the end of the build
 cache:
   directories:
+    # The following two directories are for zipkin-web, which gets dependencies via NPM
+    - $HOME/.npm
+    - zipkin-web/node_modules
```

Comment:
removed via https://github.com/openzipkin/zipkin/commit/08bd0ee6d95ced8c9f6f11c24269b64e887bc62a

thx for thinking this through!